client/hotrod/src/main/java/org/infinispan/hotrod/HotRodMutinyCaches.java

```diff
@@ -20,29 +22,30 @@ public class HotRodMutinyCaches implements MutinyCaches {
 
    @Override
    public <K, V> Uni<MutinyCache<K, V>> create(String name, CacheConfiguration cacheConfiguration) {
-      // FIXME
-      return Uni.createFrom().item(hotrod.transport.getRemoteCache(name)).map(r -> new HotRodMutinyCache<>(hotrod, (RemoteCache<K, V>) r));
+      return Uni.createFrom().completionStage(hotrod.transport.<K, V>getRemoteCache(name))
```

Comment:
Technically Uni should be completely lazy, that is the operation isn't performed until subscribed. We may want to use the Supplier completionStage method to be 100% safe.